File: hash_passwords.js
const crypto = require('crypto');

const passwords = [
    "manager123",
    "admin123",
    "member123"
];

console.log("SHA256 Password Hashes:\n");
passwords.forEach(password => {
    const hash = crypto.createHash('sha256').update(password).digest('hex');
    console.log(`"${password}" -> ${hash}`);
});

File: hash_passwords.py
import hashlib

passwords = [
    "manager123",
    "admin123",
    "member123"
]

print("SHA256 Password Hashes:\n")
for password in passwords:
    hash_result = hashlib.sha256(password.encode()).hexdigest()
    print(f'"{password}" -> {hash_result}')
